.github/workflows/build.yml

```diff
@@ -0,0 +1,41 @@
+name: Build
+
+on: [ push, pull_request, workflow_dispatch ]
+
+jobs:
+  node:
```

Comment:
This should be name build for stand alone UI. This shows as the label for the status check.

Reply:
Named to build. Sorry missed out on this.